### PR TITLE
Create a Term Entry for Python 

### PR DESCRIPTION
**Pull request by Jesuscg25**
## Description
This pull request refers to the creation of a Term Entry for Python concepts, specifically `Plotly .line()`.

The file added was an `md` file under the following path: 

`docs/content/plotly/concepts/data-visualization/terms/line/line.md`

This file contains a description and example of how to use the term in the following order: 
- `Introduction`
- `Syntax`
- `Example`
- `Conclusion`

## Issue Solved
Closes #4556 

## Type of Change
Adding a new entry
## Updating the documentation
Checklist
[✅] All writings are my own.
[✅] My entry follows the Codecademy Docs style guide.
[✅] My changes generate no new warnings.
[✅] I have performed a self-review of my own writing and code.
[✅] I have checked my entry and corrected any misspellings.
[✅] I have made corresponding changes to the documentation if needed.
[✅] I have confirmed my changes are not being pushed from my forked main branch.
[✅] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
[✅] I have linked any issues that are relevant to this PR in the Issues Solved section.

### CODE REVIEW

File: content/python/concepts/data-visualization/terms/line/line.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Below is an example demonstrating how to use the `.line()` method to create a line chart visualizing economic growth over time:
+
+```python
+import plotly.express as px
+
+# Load an example dataset
+df = px.data.gapminder().query("continent == 'Oceania'")
+
+# Create a line chart
```
Comment (mamtawardhani) [inline content/python/concepts/data-visualization/terms/line/line.md:33]:
```suggestion
# Create a line chart visualizing economic growth over time
```

File: content/python/concepts/data-visualization/terms/line/line.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+fig = px.line(df, x='year', y='gdpPercap', color='country', title='Economic Growth in Oceania')
+
+# Show the plot
+fig.show()
+
+```
+
```
Comment (mamtawardhani) [inline content/python/concepts/data-visualization/terms/line/line.md:40]: Please give the Output under the ## Output section. You can attach the image of the output using the Image block.
Please refer to the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) to add the output Image.

File: content/python/concepts/data-visualization/terms/line/line.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+In this example, the economic growth (GDP per capita) of countries in Oceania from the Gapminder dataset is plotted. The x parameter is set to 'year', and the y parameter to 'gdpPercap'. The color parameter differentiates the lines by country, providing clear visual distinctions between each country's economic trajectory.
+
+## Conclusion
+Utilizing the `.line()` method in Plotly allows for the efficient creation of dynamic and informative line charts. This tool is especially useful in data analysis for identifying trends, making comparisons across categories, and illustrating changes over time, making it a cornerstone of data visualization tasks.
```
Comment (mamtawardhani) [inline content/python/concepts/data-visualization/terms/line/line.md:44]: This can be removed, a conclusion is not needed.

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Jesuscg25!
Thank you for Contributing at Codecademy!

The entry was very well written, I've just added a few suggestions and comments that can be made for this entry. Could you please review and modify the required changes at your earliest convenience?
Thank you!

File: content/python/concepts/data-visualization/terms/line/line.md (hunk 2 above)
Comment (Jesuscg25) [reply content/python/concepts/data-visualization/terms/line/line.md:51]: Done!

File: content/python/concepts/data-visualization/terms/line/line.md (hunk 3 above)
Comment (Jesuscg25) [reply content/python/concepts/data-visualization/terms/line/line.md]: Done!

File: content/python/concepts/data-visualization/terms/line/line.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+fig = px.line(df, x='year', y='gdpPercap', color='country', title='Economic Growth in Oceania')
+
+# Show the plot
+fig.show()
+
+```
+
+In this example, the economic growth (GDP per capita) of countries in Oceania from the Gapminder dataset is plotted. The x parameter is set to 'year', and the y parameter to 'gdpPercap'. The color parameter differentiates the lines by country, providing clear visual distinctions between each country's economic trajectory.
+
+## Output
```
Comment (mamtawardhani) [inline content/python/concepts/data-visualization/terms/line/line.md:55]:
```suggestion
The code above produces the following output:
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Jesuscg25 

Just one change is required, please remove the ## Output heading, you can rather say, "the  code above produces the following output."
Thanks!

File: content/python/concepts/data-visualization/terms/line/line.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+- `data_frame`: The dataset containing the variables that will be plotted.
+- `x`: The data for the x-axis, typically time or a continuous variable.
+- `y`: The data for the y-axis, representing the measured variables.
+- `color`: (Optional) A column name in the data frame that will determine the color of the lines, useful for distinguishing different groups or categories.
+- `title`: (Optional) The title of the chart.
+- `labels`: (Optional) Custom labels for different axes or legend items.
+
+> Note: The ellipsis (...) indicates that there can be additional optional arguments beyond those listed here.
```
Comment (mamtawardhani) [inline content/python/concepts/data-visualization/terms/line/line.md:33]:
```suggestion
> **Note:** The ellipsis (...) indicates that there can be additional optional arguments beyond those listed here.
```

File: content/python/concepts/data-visualization/terms/line/line.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+```pseudo
+plotly.express.line(data_frame=None, x=None, y=None, color=None, title=None, labels=None, ...)
+```
+
+- `data_frame`: The dataset containing the variables that will be plotted.
+- `x`: The data for the x-axis, typically time or a continuous variable.
+- `y`: The data for the y-axis, representing the measured variables.
+- `color`: (Optional) A column name in the data frame that will determine the color of the lines, useful for distinguishing different groups or categories.
+- `title`: (Optional) The title of the chart.
+- `labels`: (Optional) Custom labels for different axes or legend items.
```
Comment (mamtawardhani) [inline content/python/concepts/data-visualization/terms/line/line.md:31]:
```suggestion
- `color`: A column name in the data frame that will determine the color of the lines, useful for distinguishing different groups or categories. This parameter is optional.
- `title`: The title of the chart. This parameter is optional.
- `labels`: Custom labels for different axes or legend items. This parameter is optional.
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Jesuscg25! Just two more changes are required, please review and modify them.
Thanks!

**Review by mamtawardhani — APPROVED** [review]
Good for the second review

**Review by Sriparno08 — APPROVED** [review]
Looks good 👍

**Comment by Jesuscg25** [comment]
Thank you for all your support!

**Comment by Sriparno08** [comment]
Hey @Jesuscg25, please don't close this PR as it is yet to get merged.

**Pull request merged**